PCGen-Formula/code/src/java/pcgen/base/formula/parse/SimpleNode.java

```diff
@@ -35,7 +34,7 @@ public SimpleNode(int i)
 	public SimpleNode(FormulaParser p, int i)
 	{
 		this(i);
-		parser = p;
+		FormulaParser parser = p;
```

Comment:
Again, this is derived code, I don't think we should be changing it, please remove this part of the PR